src/client/client.js

```diff
@@ -14,6 +14,26 @@ function removeEmpty(obj) {
     return obj;
 }
 
+/**
+ * getAccceptFormat returns the correct Accept header depending on the
+ * requested format.
+ * @param query
+ * @returns {string}
+ */
+function getAccceptFormat(query) {
+    if (query !== undefined && query.hasOwnProperty('format'))
+        switch(query.format) {
+            case 'msgpack':
+                return 'application/msgpack';
+            case 'json':
+                return 'application/json';
+            default:
+                return 'application/json';
+        }
+    else
+        return "application/json"
+ }
```

Comment:
tiny comment - since we're dealing with only two type, this would be sufficient.
```js
function getAccceptFormat(query) {
	if (query !== undefined && query.hasOwnProperty('format'))
		if (query.format === 'msgpack')
			return 'application/msgpack';
	return "application/json"
 }
```

Reply:
We thought of that approach, but wanted to present a solution that allowed to easily add additional types, thinking that in the future that might happen. But if there's only going to be two types then I'll change the code to yours!